### PR TITLE
Razorpay clone Addition | Issue #826

### PR DESCRIPTION
**Pull request by Shariq2003**
### Description

This PR introduces the RazorPay-Clone project, a replica of the Razorpay website created using HTML and TailwindCSS. The primary goal of this project is to practice web development skills, focusing on creating responsive, modern layouts using TailwindCSS.

The clone captures the design elements of the original Razorpay website and can be viewed on different screen sizes. This project serves as a demonstration of my proficiency in frontend development using minimal, utility-first CSS frameworks.

Closes: #826 

- **Title:** Razorpay Clone Addition 
- **Name:** Shariq
- **Idenitfy yourself:** 
GSSOC Ext

### Describe the add-ons or changes you've made 📃
Added the Above mentioned project.

### Screenshots 📷

image
image
image

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### CODE REVIEW

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, please review

1. I have readme of project to include images and videos
2. Update Readme of Frontend Projects
3. Follow Cabab Casing.

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney please review this

File: Front-end-Projects/Intermediate/Razorpay-Clone/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Razorpay-Clone/README.md:1]: https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/PROJECT-README-TEMPLATE.md

File: Front-end-Projects/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/README.md:1]: Update it in row 5

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney please check now

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, review now

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, Now i have reviewed all the changes and make the PR ready to merge.
Please review this

**Comment by Shariq2003** [comment]
@Avdhesh-Varshney sir, could u please merge the PR in order they are creating in time, as I have created this PR which has no conflicts now, but if u merge a PR that has raised after me, then there will be conflicts in my PR in readme.md file.

So please merge this

File: Front-end-Projects/Intermediate/Razorpay-Clone/Working Video.mp4
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Razorpay-Clone/Working Video.mp4:1]: Delete it not required.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Pull request closed without merging**